xml/ay_custom_user_scripts.xml

```diff
@@ -484,12 +489,13 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
          <para>
           If this is <literal>true</literal>, every single line of a shell
           script is logged. Perl scripts are run with warnings turned on.
+          For other interpreters use directly parameter in interpreter field.
          </para>
 <screen>&lt;debug config:type="boolean"&gt;true&lt;/debug&gt;</screen>
         </entry>
         <entry>
          <para>
-          Optional; default is <literal>true</literal>.
+          Optional; default is <literal>false</literal>.
```

Comment:
I've checked the code and indeed you are changing a default that the users may rely on, IMHO totally unnecessarily.

Reply:
well, for me it is really strange to have debugging enabled by default. It will also quite mess output. To be honest as autoyast is non-interactive I expect not much customers show any feedback and just let script run hidden.

Reply:
also consider now that you can specify in interpreter "sh" or "shell" and if you do not specify debug, it means that sh run just as sh and shell runs with -x...For perl situation is even more strange as "perl" run with -w and "/bin/perl" runs without it.

Reply:
maybe we can even drop that debug flag and ignore it? and just for backward compatible convert "shell" interpreter to "sh" and kill all keywords which makes it just confusing?

Reply:
If we decide to keep the new behavior, we should document that change.

Reply:
@lproven Liam we discuss it and will revert back to keep same behavior, but we would like to mark this element and also keywords interpreters as deprecated that will be removed in SLE16. Is there way how to do it in docbook?

Reply:
I don't know any specific markup for something being deprecated, but we could just add a <info> or <warning> block mentioning it. Would that be enough?